whylog_vim/output_formater/utils.py

```diff
@@ -0,0 +1,5 @@
+def to_buttons(elem_list):
```

Comment:
Please rename. I didn't understand while reading function name what could it mean. Maybe `convert_to_buttons' or `create_buttons_text` or figure out something.